frontend/lib/about/dev_settings_view.dart

```diff
@@ -179,6 +164,53 @@ class DevSettingsView extends StatelessWidget {
     );
   }
 
+  Future<void> _activateCard(BuildContext context, String base64qrcode) async {
+    final messengerState = ScaffoldMessenger.of(context);
+    final provider = Provider.of<UserCodeModel>(context, listen: false);
+    final client = GraphQLProvider.of(context).value;
+    final projectId = Configuration.of(context).projectId;
+    try {
+      final activationCode =
+          const ActivationCodeParser().parseQrCodeContent(const Base64Decoder().convert(base64qrcode));
+
+      final activationResult = await activateCode(
+        client: client,
+        projectId: projectId,
+        code: activationCode,
+        overwriteExisting: true,
+      );
+
+      if (activationResult.activationState == ActivationState.failed) {
+        throw QrCodeParseException("Code could not be activated from the server");
+      }
+      if (activationResult.activationState == ActivationState.didNotOverwriteExisting) {
+        throw QrCodeParseException("Did not overwrite existing QrCode.");
+      }
+      if (activationResult.totpSecret != null) {
+        final totpSecret = const Base64Decoder().convert(activationResult.totpSecret!);
+        final userCode = DynamicUserCode(
+          info: activationCode.info,
+          pepper: activationCode.pepper,
+          totpSecret: totpSecret,
+        );
+        provider.setCode(userCode);
+      }
+
+      messengerState.showSnackBar(
+        const SnackBar(
+          content: Text("Card activated."),
+        ),
+      );
```

Comment:
This should be translated to German + whitelabeled.
(
 Nürnberg => "Nürnberg-Pass aktiviert."
 Bayern => "Ehrenamtskarte aktiviert."
)

Reply:
This is still the dev_settings_view. I can change the text, but this should never be visible to the user.

Reply:
Agree, sorry I somehow overlooked that this is only the dev settings view...  Should not be whitelabeled :D
Also, probably not important to change the text here...